src/api/services/format-date-service.ts

```diff
@@ -4,19 +4,23 @@ import moment from 'moment';
 @Injectable()
 // TODO: probably a good idea to create a interface for this, will revisit this when we defined the architecture for the lang
 export class FormatDate {
+    private m;
```

Comment:
Question: shouldn't init it on the constructor or at least check in the getLang if this.m is undefined?